javascript/script.js

```diff
@@ -0,0 +1,10 @@
+// .script
+var list = document.getElementByID('list'),
+	add = document.getElementByID('addElem');
+
+add.addEventListener('click', function() {
+	document.getElementByID('list').innerHTML += '<li>item</li>';
```

Comment:
nie pobieraj na nowo elementu po jego id tylko wykorzystaj wyżej zadeklarowaną zmienną, do tego `'<li>item' + liczba + '</li>'` wystarczy żeby generować html typu  `<li>item 1</li>`, `<li>item 2</li>` itp - kwestia tylko czym zastąpić to "liczba". `document.getElementsByTagName()` zwraca tablicę i to jej długość może nam posłużyć za ów liczbę.